content/en/developers/events/dogstatsd.md

```diff
@@ -171,8 +171,10 @@ $statsd->event('An error occurred.',
   );
 {{< /code-block >}}
 
+{{< markdownify >}}
```

Comment:
need to use this new shortcode `markdownify` because we are now using `{{< tab "PHP" >}}` the `<` syntax which will not render inner content as markdown. Links also won't render using [link text][3] syntax with the link at the bottom of the page. Need to add the link inline. The format-links pre-commit hook tries to change this, so will have to commit code with --no-verify.